README.md

```diff
@@ -69,6 +71,7 @@ The following types should be used as applicable `rel` types in the
 
 | Type    | Description |
 | ------- | ----------- |
+| author | An ORCID link SHOULD be added to the links section for the author(s) referenced by the `sci:orcids` property with the `rel` type `author`. (see [rel type author](https://html.spec.whatwg.org/multipage/links.html#link-type-author)) |
```

Comment:
Maybe separate ror and orcid relation types? We can probably just name the relation types ror and orcid...

Reply:
👍 I wasn't sure if relation types were managed by a standards organization. I'll use `ror` and `orcid`.